pkgs/by-name/mi/mignon-icon-theme/package.nix

```
@@ -0,0 +1,70 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  gtk3,
+  kdePackages,
+  hicolor-icon-theme,
+  adwaita-icon-theme,
+  mint-y-icons,
+}:
+
+stdenvNoCC.mkDerivation (finalAttr: {
+  pname = "mignon-icon-theme";
+  version = "unstable-2025-09-05";
```

you must start the version string with a numeric version, or a zero, if there's none
```suggestion
  version = "0-unstable-2025-09-05";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning